### PR TITLE
[Page] + load children of the root in tree as default

### PR DESCRIPTION
**Pull request by fkroockmann**
Currently, the tree show just a root not a children.

### CODE REVIEW

File: src/tb/apps/page/views/page.view.tree.js [hunk 1]
```diff
@@ -162,19 +163,11 @@ define(
...
-                } else if (event.namespace === 'click') {
-                    target = event.node.parent;
-                }
-
-                if (target !== undefined) {
-                    target.range_total = response.getRangeTotal();
-                    target.range_to = response.getRangeTo();
-                    target.range_from = response.getRangeFrom();
+            updateLimit: function (node, response) {
+                if (node !== undefined) {
```
Comment (hbaptiste) [inline src/tb/apps/page/views/page.view.tree.js]: Why not return if node is undefined?

Comment (fkroockmann) [inline src/tb/apps/page/views/page.view.tree.js]: I don't understand, can you explain ?

Comment (hbaptiste) [inline src/tb/apps/page/views/page.view.tree.js]: I propose to change  
if (node !== undefined) {
to if(!node) {return;}

Comment (fkroockmann) [inline src/tb/apps/page/views/page.view.tree.js]: The function is too small and i prefer to use a condition for doing a treatment.

**Comment by fkroockmann** [comment]
Thanks for review @hbaptiste :)

**Pull request merged**